app/main.py

```diff
@@ -1,25 +1,95 @@
 from abc import ABC
 
 
+class Visitor:
+    def __init__(
+            self,
+            name: str,
+            age: int,
+            weight: int,
+            height: int
+    ) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
+
+
 class IntegerRange:
-    pass
+    def __init__(
+            self,
+            min_amount: int,
+            max_amount: int
+    ) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
 
+    def __set_name__(
+            self,
+            owner: type,
+            name: str
+    ) -> None:
+        self.protected_name = f"_{name}"
 
-class Visitor:
-   pass
+    def __get__(
+            self,
+            instance: Visitor,
+            owner: type
```

Comment:
invalid typehint for 'owner'

Reply:
done, thanks for comment
now owner typehin is class itself, not an instance